docs/usage/models.md

```diff
@@ -798,12 +801,12 @@ except ValidationError as e:
     """
 ```
 
-For many use cases `TypeAdapter` can replace BaseModels with a `__root__` field in Pydantic V1.
+For many use cases, `TypeAdapter` can replace `BaseModel`s with a `__root__` field in Pydantic V1.
```

Comment:
```suggestion
For some use cases, `TypeAdapter` can replace `BaseModel`s with a `__root__` field in Pydantic V1.
```
I think saying *many* here is a bit too strong; most of the usage I saw of `__root__` when searching github involved the use of methods, which is what would require `RootModel`. I think saying "For some use cases" is fine though, regardless of the actual distribution.

It may make sense to point people at `RootModel` here since we are referencing `__root__` (as this is the way to handle the other usages of `__root__` from V1).